zds/utils/templatetags/emarkdown.py

```diff
@@ -22,35 +21,36 @@
 # Constant strings
 MD_PARSING_ERROR = _(u'Une erreur est survenue dans la génération de texte Markdown. Veuillez rapporter le bug.')
 
-markdown_client = zerorpc.Client(heartbeat=5)
-markdown_client.connect("tcp://127.0.0.1:24242")
-
 
 def render_markdown(md_input, **kwargs):
     """
     Render a markdown string.
     """
+    attempts = kwargs.get('attempts', 0)
     inline = kwargs.get('inline', False) is True
     is_latex = kwargs.pop('is_latex', False) is True
-    markdown_compiler = markdown_client.toHTML if not is_latex else markdown_client.toLatex
+    endpoint = '/html' if not is_latex else '/latex'
```

Comment:
Ouais peut-être, à voir.

Je me serais pas permis un `x = y if not not_foo else foo`, ça ce serait une double négation et ce serait pénible à lire. J'ai fait ça comme ça ici parce que clairement 99% des requêtes iront sur l'endpoint `/html` et que je préfère noter que sauf exception c'est là qu'on va.

Je crois que je préférerais

```python
endoint = '/html'
if is_latex:
    endpoint = '/latex'
```

à ta solution, pour continuer à véhiculer cette impression de valeur par défaut.